Darwin/flash

```diff
@@ -97,8 +97,8 @@ filename=$(basename "${image}")
 extension="${filename##*.}"
 filename="${filename%.*}"
 
-if [ -f "/tmp/${filename}.${extension}" ]; then
-  image=/tmp/${filename}.${extension}
+if [ -f "/tmp/${filename}" ]; then
```

Comment:
This reverts https://github.com/surround-io/flash/commit/859e0cbb800e6d097faa5ddf39bbd18f27973553
I think I've changed that while building new zipped image files locally and tested and build and tested again.